lib/mayaUsd/base/tokens.h

```diff
@@ -68,6 +68,30 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 TF_DECLARE_PUBLIC_TOKENS(MayaUsdOptionVars, MAYAUSD_CORE_PUBLIC, MAYA_USD_OPTIONVAR_TOKENS);
 
+// Tokens that are used as metadata on prim and attributes in MayaUSD
+//
+// clang-format off
+#define MAYA_USD_METADATA_TOKENS                         \
+    /* Selectability metadata to be used on prims     */ \
+    ((Selectability, "mayaSelectability"))               \
+    /* Locking attribute metadata. A locked attribute */ \
+    /* value cannot be changed.                       */ \
+    ((Lock, "mayaLock"))                                 \
+    /* Metadata value to inherit the value from a     */ \
+    /* parent prim. Used in selectability.            */ \
+    ((Inherit, "inherit"))                               \
+    /* Metadata value to turn on or off a feature.    */ \
+    /* Used in selectability and lock, for example.   */ \
+    ((On, "on"))                                         \
```

Comment:
I think it would make it more awkward to use since the metadata and its value would live in two separate objects in scripting. OTOH, the same token can live in multiple places, so if they are needed elsewhere than just for metadata, they could also be available there. Currently, even though I agree they are general terms, they are only used in metadata values.

Reply:
Would the simple name "Tokens" would do? short enough when used in scripting and avoids polluting the global namespace of the Python module.

Reply:
If you're suggesting we keep the metadata tokens separate, and create a new MayaUsd.Tokens namespace in Python, then yes that sounds good to me.